packages/api/src/external/aws/address.ts

```diff
@@ -4,24 +4,25 @@ import {
   makeLocationClient,
 } from "@metriport/core/external/aws/location";
 import { Config } from "../../shared/config";
+import * as AWS from "aws-sdk";
 import { Coordinates } from "@metriport/core/external/aws/location";
 
+const indexName = Config.getPlaceIndexName();
+const awsRegion = Config.isProdEnv() ? Config.getPlaceIndexProdRegion() : Config.getAWSRegion();
```

Comment:
Why don't we just have one env var that configures the place index region? We don't need this complexity to have a separate prod env var, and then logic to deal with it

Reply:
Oh, good point!